includes/blast_ui.form_advanced_options.inc

```diff
@@ -76,17 +68,6 @@ function blast_ui_blastn_advanced_options_form(&$form, $form_state) {
     '#description' => t('The length of the seed that initiates an alignment'),
   );
 
-  /*eksc: remove this as it is either the same as max_target_seqs, or miss-implemented
-         as culling_limit, which is something else entirely
-    $form['ALG']['GParam']['qRange'] = array(
```

Comment:
qrange shows up in a few submissions.  Looks like we just want to kill this parameter?  It is a valid parameter on ncbi.  DOn't get me wrong i don't care i just want the submits to pass without errors...